example/src/stories/BottomSheet.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import type {ComponentMeta, ComponentStory} from '@storybook/react'
+import React, {useRef} from 'react'
+import {Text, TextInput} from 'react-native'
+import {useSafeAreaInsets} from 'react-native-safe-area-context'
+import {BottomSheet, Button} from 'rn-base-component'
+import type {BottomSheetMethods} from 'src/components/BottomSheet/types'
+
+export default {
+  title: 'components/BottomSheet',
+  component: BottomSheet,
+} as ComponentMeta<typeof BottomSheet>
+
+export const Basic: ComponentStory<typeof BottomSheet> = args => {
+  const ref = useRef<BottomSheetMethods>(null)
+  const insets = useSafeAreaInsets()
+
+  return (
+    <>
+      <Button text="wefiowejfi" onPress={() => ref?.current?.open?.()} />
```

Comment:
I know the text **wefiowejfi** is just dummy text to display storybook but can we change the text for more clearly and and relate to bottom sheet?

Reply:
oke, updated